app/decorators/video_decorator.rb

```diff
@@ -0,0 +1,12 @@
+class VideoDecorator < Draper::Decorator
+  delegate_all
+
+  def show_rating
+    average_rating ? "#{show_average_rating}/5.0" : "N/A"
+  end
+
+  private
+  def show_average_rating
```

Comment:
I thought it may confused when you used `video.rating` on a model class, it may mean you will get a pure data or a string for presentation. But maybe it is not a good naming neither. do you has any suggestion? :)